packages/docs/source/form/date/components/date-field.md

```diff
@@ -44,4 +44,13 @@ Extends [Date Props](/form/date/components/date/#props).
 The name of the field. Will be the key of the selected date that comes through in the values of the `onSubmit` callback.
 
 ### `label?: string`
-The text that renders inside the `Label` above the input.
\ No newline at end of file
+The text that renders inside the `Label` above the input.
+
+### `labelClass?: string`
+The name of the class for the label. Will be passed to the `className` prop of the label in the field.
+
+### `labelHidden?: boolean`
+Used to control if the label is displayed. When set to `true`, the `<Label>` in the field won't be rendered.
```

Comment:
I would reword this to say the field will not be visible.
It is still rendered so that assistive technology (such as screen readers) know what the field's label is.
Same for the range docs